main/lsp/LSPTypechecker.cc

```diff
@@ -450,6 +450,13 @@ bool LSPTypechecker::runSlowPath(LSPFileUpdates updates, WorkerPool &workers, bo
             return;
         }
 
+        // Test-only hook: Stall for as long as `slowPathBlocked` is set.
+        {
+            absl::MutexLock lck(&slowPathBlockedMutex);
+            slowPathBlockedMutex.Await(absl::Condition(
+                +[](bool *slowPathBlocked) -> bool { return !*slowPathBlocked; }, &slowPathBlocked));
```

Comment:
Looking at the abseil sources, it looks like this can just be:

```suggestion
            slowPathBlockedMutex.Await(absl::Condition(&slowPathBlocked));
```

or did you try that already and found that it didn't work?

Reply:
Ooh, no, let me give it a shot!

Reply:
Ah, unfortunately this doesn't quite work because the condition we're waiting for is that `slowPathBlocked` is false, not true. I suppose we could change this internally, but on balance I think the extra complexity here is worth keeping "true means blocked" semantics on the surface and in the implementation. (Glad to change in a later PR if there's objections, though.)